config.template.yml

```
@@ -595,6 +595,10 @@ notifier:
     ## Enables additional debug messages.
     # enable_client_debug_messages: false
 
+    ## SECURITY NOTICE: It's not recommended to change this option, and highly discouraged to have it above 8 for
```

```suggestion
    ## SECURITY NOTICE: It's not recommended to change this option, and highly discouraged to have it below 8 for
```

I think this is what you're meaning to say here?